web/src/api/generated/.openapi-generator/FILES

```diff
@@ -9,12 +9,8 @@ common.ts
 configuration.ts
```

Comment:
✨ Here's a tip ✨
Check references and implementations of the generated types, to see if they're actually useful. Many of the interfaces in this branch aren't being used at all.